### DISCUSSION TITLE
Asus Zenwatch 2 - Standalone

### DISCUSSION THREAD

**Discussion by Philippe-Tome** [NightscoutFoundation/xDrip#2356] [Q&A]
Hello there,

I am thinking of buying the Asus Zenwatch 2 in order to make it as a standalone BG monitor.

* First question is is this [page](https://github.com/jamorham/xDrip-plus/blob/master/Documentation/WatchGuide.md#choosing-a-watch) still accurate regarding the support of this watch and does it work with a G6 sensor?

* Second question is if this watch works with my G6 sensor, can it work at the same time with a Tandem Tslim X2 pump without creating any conflicts?

* Third question: can I also have the phone connected to the G6, so in total it would be 3 devices: the pump, smartwatch, smartphone (in order of importance for me)?

My pump is used with Control IQ and to collect the data to be shared with my endo, so it is very important that its connection stays reliable.

Thank you.

**Comment by Navid200** [comment]
I don't think xDrip has been modified to disable the support of any particular watch.

A G6 transmitter has only 2 slots.  
The t:slim pump uses one of those two slots.
You can then have one other device act as a direct collector.
So, if you want, you can have xDrip collect.

Now, if you set up the watch with wearOS, everything should be fine.  But, the watch cannot also be a stand-alone collector at the same time.

**Reply by Philippe-Tome** [reply]
Thanks for your reply.

- So this firmware will work with a G6 transmitter, is that right? Because the docs only talks about G5.
- On your last point, I don't fully understand. Despite having 2 slots on the transmitter, only one device can be a stand-alone collector. Is that right? And since in my case the pump must stay a standalone collector device I can't make the watch as a standalone device too?

Thanks.

**Reply by Navid200** [reply]
If G6 didn't work before with a watch, it may not work now either.  I'm sorry I'm not experienced with watches.  I may have misunderstood your question.

There are 2 slots.  Each slot can accommodate a collector.  The t:slim pump is a collector itself.  That leaves one more slot for either your phone or for your watch if it is going to be a standalone collector.
But, if your watch is wearOS, it will receive its readings from the phone.  So, it will not work if you leave the phone behind.

**Reply by Philippe-Tome** [reply]
Thanks. So if I understand well the wearOs patch is supposed to allow the watch to work as a standalone but we don't know for sure if it works with G6 sensors, is that correct?

**Comment by Navid200** [comment]
@psonnera

**Comment by psonnera** [comment]
I believe it should work if the model is WI501Q or WI502Q but not sure of what's available now, if you bought it second hand or it's an old watch I'd give it a go. Make sure you have all recovery firmware handy if things go wrong.

@Navid200 correct me if I'm wrong but t:slim uses the non-phone slot and if xDrip+ connects to the phone slot, the watch should be able to connect to that same slot when put in standalone.

This of course means no Dexcom app on the phone.

**Reply by Navid200** [reply]
I don't know how a watch that uses wearOS becomes standalone.
But, if it disables xDrip itslef so that xDrip stops communicating directly with the transmitter, then, the watch should be able to use it.

**Reply by psonnera** [reply]
@Navid200 xDrip+ stops its collector service and enables it on the watch. If xDrip+ is connected to the sensor, the watch will connect to the sensor. The issue is if the watch wear OS can handle correctly the bluetooth connection (this is why there are patches to fix these watches bugs). My only wonder is how well is the Wear code changed to match the main xDrip+ code whenever this second one is modified.

**Reply by Philippe-Tome** [reply]
Thanks for all your inputs. If I understand well it should technically be ok if I just want to use the watch and the pump but we are not 100% sure regarding the watch firmware updates etc...
